examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
> If I apply proj first, then filter (with picks), it is different from filtering (with picks) and then applying proj? Is that correct?
Yep - if you do proj first, then filter, then it will work the way you'd imagine. If you filter first then proj, then you'll get some of the non-filtered channels mixed in with the result of proj.

> What about when there is no picks and you have all channels? Does the order in which you apply proj and filter matter?

It shouldn't - to my knowledge there is no "picks" available for _any_ of the projection code. It always uses the full projection matrix for the channels that the projection applies to, I think with the bad channels taken out, but it doesn't let you select arbitrary subsets.

> According to your reasoning, it shouldn't matter because one is temporal and the other is spatial filtering.

I think the fact that it's a spatial filter is precisely the problem. It's a spatial filter as in "it takes signals distributed in space and mixes them together". If some of those signals are filtered and others aren't then the effects of filtering will get washed out.

> Also, we need additional tests for these corner cases.

I'm speaking from ignorance here since I haven't done SSP before, but to me seems that since `picks` was never built into any of the projection code, then it was never intended for the user to subset particular rows/columns when doing the projection. So the original version of the compute_XXX_psd functions was doing stuff that shouldn't have been allowed (subsetting the projection code according to `picks`). Now that we're using `apply_proj`, it will always use the entire projection matrix, which is what should happen.

In terms of warnings etc, I'd like either:

1. A warning within the projection code that says something to the effect of "applying projections mixes channel information, any operations done on a subset of channels will be nullified"
2. Create an example that illustrates this problem (or a tutorial)

To me it's not an edge case or a bug, so much as a case of making clear what applying the SSP projections does - namely, that it mixes the information for all of the channels together.

Hopefully none of that was completely incorrect :)